src/System.Windows.Forms/src/System/Windows/Forms/TabControl.cs

```diff
@@ -1666,7 +1667,11 @@ protected void RemoveAll()
         {
             Controls.Clear();
 
-            SendMessage((int)ComCtl32.TCM.DELETEALLITEMS, 0, 0);
+            if (IsHandleCreated)
+            {
+                User32.SendMessageW(this, ((User32.WindowMessage)TCM.DELETEALLITEMS), IntPtr.Zero, IntPtr.Zero);
```

Comment:
I think we have an overload
```suggestion
                User32.SendMessageW(this, ((User32.WindowMessage)TCM.DELETEALLITEMS));
```

Reply:
nope sadly